### DISCUSSION TITLE
[灌水]

### DISCUSSION THREAD

**Discussion by KELERSAMA** [share121/inter-knot#5683] [公告]
![Image_184101666304820](https://github.com/user-attachments/assets/a483f72f-66b4-48b8-84da-3d902e6e7f61)

![Image_184099014701904](https://github.com/user-attachments/assets/2a8e0b12-88a7-42ad-b17a-ffa225ced708)

**Comment by hot-ice-seele** [comment]
🐟

**Comment by 123acnl** [comment]
怎么发帖子呀楼主

**Reply by hot-ice-seele** [reply]
首页右下角可以写

**Comment by 123acnl** [comment]
![Screenshot_20241118_000613](https://github.com/user-attachments/assets/63be376f-d7f7-4dfd-833a-46f878d8eee8)
出来的是投票

**Reply by hot-ice-seele** [reply]
![Screenshot_20241118_004917.jpg](https://github.com/user-attachments/assets/55266f6c-340b-4c75-b9e0-2ff662b68b23)

蓝框内可以写字

**Comment by 123acnl** [comment]
还真可以
